### DISCUSSION TITLE
Demo failed to run

### DISCUSSION THREAD

**Discussion by hehetiger** [ant-design-blazor/ant-design-blazor#4128] [Q&A]
I downloaded the master and when I tried to decompress and regenerate the solution, it prompted that the command "npm run build: lib" had exited with code 1.Why did this error occur and how can it be resolved?

**Comment by ElderJames** [comment]
Thanks for contacting us @hehetiger , please follow the development guide in README. You need to build the solution first.

```
$ git clone https://github.com/ant-design-blazor/ant-design-blazor.git
$ cd ant-design-blazor
$ npm install
$ dotnet build ./site/AntDesign.Docs.Build/AntDesign.Docs.Build.csproj
```

**Reply by hehetiger** [reply]
When i try 'git clone https://github.com/ant-design-blazor/ant-design-blazor.git',it prompted that the command ' Failed to connect to github.com port 443'.
Now I am trying to download the zip code directly, unzip it, and execute 'npm install' and 'dotnet build ./site/AntDesign.Docs.Build/AntDesign.Docs.Build.csproj’
When executing dotnet build, prompt 'error MSB3073: Command' npm run '
Build: lib "has exited with code 1
![企业微信截图_20240826133831](https://github.com/user-attachments/assets/3449ee71-51f7-4dd0-8a5a-22c197e515e5)
